src/components/inputtools/TextStyle.vue

```diff
@@ -1,40 +1,36 @@
-                 data-code="02" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-green"
-                 data-code="03" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-red"
-                 data-code="04" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-brown"
-                 data-code="05" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-purple"
-                 data-code="06" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-orange"
-                 data-code="07" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-yellow"
-                 data-code="08" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-green"
-                 data-code="09" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-cyan"
-                 data-code="10" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-cyan"
-                 data-code="11" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-blue"
-                 data-code="12" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-pink"
-                 data-code="13" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-grey"
-                 data-code="14" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour irc-bg-colour-light-grey"
-                 data-code="15" @click="onColourClick"/>
-            <div class="kiwi-inputtools-colours-colour kiwi-inputtools-colours-reset"
-                 @click="onResetClick"><i class="fa fa-ban" aria-hidden="true"/></div>
+            <div :class="{active: !showModernPalette}"
+                 class="kiwi-inputtool-colour-button"
+                 @click="showModernPalette = !showModernPalette"
+            >
+                <i v-if="!showModernPalette"
+                   class="fa fa-plus-circle" aria-hidden="true" />
+                <i v-if="showModernPalette"
+                   class="fa fa-minus-circle" aria-hidden="true" />
+            </div>
+
+            <div v-if="!showModernPalette"
+                 class="kiwi-inputtools-colours">
+                <div v-for="n in 15"
+                     :key="n"
+                     :class="['irc-bg-colour-' + n]"
+                     :data-code="[n]"
+                     class="kiwi-inputtools-colours-colour"
+                     @click="onColourClick" />
+            </div>
+
+            <div v-else
+                 class="kiwi-inputtools-colours">
+                <div v-for="n in 96"
```

Comment:
Just curious, is this meant to be 96 or 98? 98 is where the extended list of colours stops.

Reply:
98, I updated it ahah